public/src/main/java/com/revenuecat/purchases/errors.kt

```diff
@@ -34,7 +34,7 @@ enum class PurchasesErrorCode(val code: Int, val description: String) {
     MissingReceiptFileError(9, "The receipt is missing."),
     NetworkError(10, "Error performing request."),
     InvalidCredentialsError(11, "There was a credentials issue. Check the underlying error for more details."),
-    UnexpectedBackendResponseError(12, "Received malformed response from the backend."),
+    UnexpectedBackendResponseError(12, "Received unexpected response from the backend."),
```

Comment:
This is only used when the backend returns a 7110 error code (Internal server error). I would expect that on 5xx error codes, the backend wouldn't really return anything in the body though. In which case, `UnknownBackendError`, `There was an unknown backend error.` is used instead.

In any case, the response from the backend in this case it's not actually malformed, so changed it to `unexpected` instead.